File: apps/backend/Music.Core/Enums/Album.cs
using System.Text.Json.Serialization;

namespace Music.Core.Enums;

[JsonConverter(typeof(JsonStringEnumConverter))]
public enum AlbumType
{
    Album,
    Single,
    Compilation,
    Live,
    Soundtrack,
    Remix,
    Other = 99
}

File: apps/web/src/enums/album.ts
import type { components } from "@/data/APIschema";

type AlbumType = components["schemas"]["AlbumType"];

export const ALBUM_TYPES: { value: AlbumType; label: string }[] = [
	{ value: "Album", label: "Album" },
	{ value: "Single", label: "Single" },
	{ value: "Compilation", label: "Compilation" },
	{ value: "Live", label: "Live" },
	{ value: "Remix", label: "Remix" },
	{ value: "Soundtrack", label: "Soundtrack" },
	{ value: "Other", label: "Other" },
];
